src/frontend/services/WishlistServices/removeFromWishlist.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+export const removeFromWishlist = async (authToken, product, setWishlist) => {
+  try {
+    const request = await axios.delete(`/api/user/wishlist/${product._id}`, {
+      headers: {
+        authorization: authToken,
+      },
+    });
+    setWishlist(request.data.wishlist);
```

Comment:
Suggestion: variable name should be response instead of request.

Reply:
I think we req to sever for delete some item ?